### PR TITLE
So much better

### PR DESCRIPTION
**Pull request by marksost**

### CODE REVIEW

**Comment by lslaslo** [comment]
👍

**Comment by marksost** [comment]
@peterghintz plz

File: assets/css/style.css [hunk 1]
```diff
@@ -79,7 +79,8 @@ body {
   opacity: 0; }
   #thumbs img {
     cursor: pointer;
-    margin: 0 5px 0 0; }
+    margin: 0 5px 0 0;
+    rotate: 90deg; }
```
Comment (DanHoerst) [inline assets/css/style.css]: Can we have this be a random int in `45` degree intervals?

**Comment by DanHoerst** [comment]
I'll just go F myself

**Comment by aj-may** [comment]
@marksost plz update this per @peterghintz feedback

**Comment by aj-may** [comment]
also needs rebase

**Comment by marksost** [comment]
@aj-may @peterghintz all comments addressed

**Comment by aj-may** [comment]
I feel like this could cause some usability issues.  Can you make it so this is triggered by the konami code?

**Comment by marksost** [comment]
If my contributions are not wanted in this codebase I will take my work to andywatching.com instead!!

**Pull request closed without merging**